.env

```diff
@@ -1,3 +1,4 @@
+ENVIRONMENT='production'
```

Comment:
nit: `NODE_ENV` is [convention](https://nodejs.dev/learn/nodejs-the-difference-between-development-and-production)